corehq/apps/hqadmin/templates/hqadmin/bootstrap3/hqadmin_base_filters.html

```diff
@@ -0,0 +1,36 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+<div class="hq-accordion-control">
```

Comment:
Hrm, it's odd to me to have files that live in a bootstrap3 directory but aren't migrated. Don't think it's a huge deal, but a little confusing.

Yeah, I only read the combined diff, definitely feel free to ignore comments on stuff you didn't touch that would be distracting to deal with right now.